### PR TITLE
topology2: add rt713_rt1316, no sdw_dmic

### PR DESCRIPTION
**Pull request by macchian**
This is additional HW board: No SDW_DMIC configuration

SDW0: rt713 Jack codec
SDW1: rt1316 smart amp l
SDW2: rt1316 smart amp r

### CODE REVIEW

File: tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake [hunk 1]
```diff
@@ -39,6 +39,11 @@ SDW_DMIC_STREAM=SDW0-Capture"
 SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,\
 SDW_DMIC_STREAM=Capture-SmartMic,SDW_JACK_OUT_STREAM=Playback-SimpleJack,\
 SDW_JACK_IN_STREAM=Capture-SimpleJack"
+
+# Jack codec + SmartAmp topology. No SDW_DMIC connection
+"cavs-sdw\;sof-mtl-rt713-l0-rt1316-l12\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=2,\
+SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
```
Comment (bardliao) [inline tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake:46]: @macchian We should use the default stream name. Please see https://github.com/thesofproject/sof/pull/8305 for your reference.

Comment (macchian) [reply tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake]: @bardliao , Thanks for pointing out!

**Review by bardliao — APPROVED** [review]

File: tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake [hunk 2]
```diff
@@ -39,6 +39,10 @@ SDW_DMIC_STREAM=SDW0-Capture"
 SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,\
 SDW_DMIC_STREAM=Capture-SmartMic,SDW_JACK_OUT_STREAM=Playback-SimpleJack,\
 SDW_JACK_IN_STREAM=Capture-SimpleJack"
+
+# Jack codec + SmartAmp topology. No SDW_DMIC connection
```
Comment (kv2019i) [inline tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake:43]: @bardliao @plbossart Do we have some convention to expose dmic/no-dmic in the tplg filename. I guess now we have not had any conflicts where we'd need the same configuration with and without dmic, so I guess this is ok, but how will this scale? Add "-dmic" when/if such cames arises?

**Review by kv2019i — APPROVED** [review]

File: tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake (hunk 2 above)
Comment (plbossart) [reply tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake:43]: if there is no rt715 in the name then there's no dmic IIRC. @bardliao can you confirm?

Comment (bardliao) [reply tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake:43]: > if there is no rt715 in the name then there's no dmic IIRC. @bardliao can you confirm?

Right, but I guess what @kv2019i mean is no PCH DMIC. I.e. sof-mtl-rt713-l0-rt1316-l12 with or without PCH DMIC.

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@macchian some conflicts.

**Comment by kv2019i** [comment]
This looks ready to go, just need a clean run of the mandatory CI checks and we can merge.

**Pull request merged**